pkg/granted/sso.go

```diff
@@ -199,6 +201,57 @@ var PopulateCommand = cli.Command{
 	},
 }
 
+var LoginCommand = cli.Command{
+	Name:  "login",
+	Usage: "Log in via AWS SSO interactive credential process",
+	Flags: []cli.Flag{
+		&cli.StringFlag{Name: "sso-region", Usage: "Specify the SSO region"},
+		&cli.StringFlag{Name: "sso-start-url", Usage: "Specify the SSO start url"},
+	},
+	Action: func(c *cli.Context) error {
+		ctx := c.Context
+		ssoStartUrl := c.String("sso-start-url")
+
+		if ssoStartUrl == "" {
+			in1 := survey.Input{Message: "SSO Start URL"}
+			err := testable.AskOne(&in1, &ssoStartUrl)
+			if err != nil {
+				return err
+			}
+		}
+
+		ssoRegion := c.String("sso-region")
+		if ssoRegion == "" {
+			in2 := survey.Input{Message: "Region"}
+			err := testable.AskOne(&in2, &ssoRegion)
+			if err != nil {
+				return err
+			}
+		}
+
+		cfg := aws.NewConfig()
+		cfg.Region = ssoRegion
+
+		secureSSOTokenStorage := securestorage.NewSecureSSOTokenStorage()
+		cachedToken := secureSSOTokenStorage.GetValidSSOToken(ssoStartUrl)
+
+		if cachedToken == nil {
+			newSSOToken, err := cfaws.SSODeviceCodeFlowFromStartUrl(ctx, *cfg, ssoStartUrl)
+			if err != nil {
+				return err
+			}
+
+			secureSSOTokenStorage.StoreSSOToken(ssoStartUrl, *newSSOToken)
+		} else {
+			clio.Info("Cached token found. Aborting SSO authentication flow.")
```

Comment:
If a user wants to refresh their token because they are having login issues this might cause an undesired effect - I think will be easier just to remove checking the cache and always save a new token here.